package.json

```diff
@@ -1,17 +1,42 @@
 {
   "name": "hijax",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
This code change didn't add functionality to the hijax library itself, so it's not a minor version update. Not 100% sure what it should be, but probably a patch aka fixing a bug where the bug is that it doesn't have license info? Thoughts